pypit/data/settings/settings.kast_red

```diff
@@ -1,3 +1,32 @@
+# New Hamamatsu detector
```

Comment:
I think we can delete this preamble... it's already in the docs (I think). It should also be removed from `settings.kast_red_ret`, as well as the archived versions of these files.